app/models/vote.rb

```diff
@@ -19,16 +19,21 @@ class Vote < ActiveRecord::Base
   belongs_to :user
 
   # scopes
-  default_scope { order(:updated_at => :desc) }
+  default_scope { by_last_updated }
```

Comment:
I always consider `default_scope` to be a configuration setting rather than an actual definition, so I avoid putting scope definitions in it.